src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -226,6 +249,22 @@ public void startTransportService(TransportService transportService) {
             (request, channel, task) -> channel.sendResponse(handlePluginsRequest(request))
         );
 
+        transportService.registerRequestHandler(
+            ExtensionsOrchestrator.REQUEST_OPENSEARCH_NAMED_WRITEABLE_REGISTRY,
+            ThreadPool.Names.GENERIC,
+            false,
+            false,
+            OpenSearchRequest::new,
+            (request, channel, task) -> channel.sendResponse(handleOpenSearchRequest(request))
+        );
+
+        transportService.registerRequestHandler(
```

Comment:
This is another overriden method you are using here. It's missing `forceExecution` and `canTripCircuitBreaker`

Reply:
Great catch, I'll make that change and update the corresponding test